README.md

```diff
@@ -36,6 +36,10 @@ export default connect((props, ref) => ({
 }))(Counter)
 ```
 
+## Test for yourself on Codepen.io
+- [Get started](https://codepen.io/einarlove/pen/peoMbp?editors=1010)
+- [Counter Example](https://codepen.io/einarlove/pen/RpwXGP?editors=1010)
+
```

Comment:
Seems strange that we're linking to something that doesn't work out of the box. Maybe start with the counter example and rename "Get started" to "Getting started template" or something?